cypress/e2e/memberships/createItemMembership.cy.ts

```diff
@@ -69,7 +73,71 @@ describe('Create Membership', () => {
     cy.get(`#${SHARE_ITEM_EMAIL_INPUT_ID}`).should('be.empty');
   });
 
-  it('cannot share item twice', () => {
+  it('open share modal, cancel and sharing reset content', () => {
+    cy.setUpApi({ items: ITEMS, members: Object.values(MEMBERS) });
+
+    // go to children item
+    const { id } = FOLDER;
+    cy.visit(buildItemSharePath(id));
+
+    // open modal and cancel
+    cy.get(buildDataCyWrapper(SHARE_BUTTON_SELECTOR)).click();
+    cy.get(buildDataCyWrapper(SHARE_ITEM_CANCEL_BUTTON_CY)).click();
+    cy.get(`#${SHARE_ITEM_SHARE_BUTTON_ID}`).should('not.exist');
+
+    // share
+    const member = MEMBERS.FANNY;
+    const permission = PermissionLevel.Admin;
+    cy.fillShareForm({
+      email: member.email,
+      permission,
+      selector: `#${CREATE_MEMBERSHIP_FORM_ID}`,
+    });
+
+    // check that fields are reset if reopen modal
+    cy.get(buildDataCyWrapper(SHARE_BUTTON_SELECTOR)).click();
+    cy.get(`#${SHARE_ITEM_EMAIL_INPUT_ID}`).should('be.empty');
+    cy.get(`.${ITEM_MEMBERSHIP_PERMISSION_SELECT_CLASS} input`).should(
+      'have.value',
+      PermissionLevel.Read,
+    );
+  });
+
+  it('share item with new admin by pressing enter', () => {
+    cy.setUpApi({ items: ITEMS, members: Object.values(MEMBERS) });
+
+    // go to children item
+    const { id } = FOLDER;
+    cy.visit(buildItemPath(id));
+
+    // share
+    const member = MEMBERS.FANNY;
+    const permission = PermissionLevel.Admin;
+    shareItem({
+      id,
+      email: `${member.email}{Enter}`,
+      permission,
+      submit: false,
+    });
```

Comment:
I'm wondering if we shouldn't press the enter after calling share item? In this way, it's clearer to see where and when the enter is done. What do you think?